.travis.yml

```diff
@@ -14,4 +14,5 @@ before_install:
   - npm i mongodb-download@2.2.3
 script:
   - npm run test:lint
-  - npm run test
\ No newline at end of file
+  - npm run test:coverage
+  - npm run report:coverage
```

Comment:
just an easy way to report cov? or more to it?

Reply:
its the same pattern as jyson, just gives you npm scrips that ci can run to report coverage